### PR TITLE
Fix build.sh to work in Jenkins environment

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
[test]

**Comment by mfojtik** [comment]
@jhadvig @bparees can one of you review/merge this? ;-) I think other PR's were merged already.

File: Makefile [hunk 1]
```diff
@@ -8,9 +8,9 @@ endif
 
 .PHONY: build
 build:
-	hack/build.sh $(OS)
+	SKIP_SQUASH=$(SKIP_SQUASH) hack/build.sh $(OS)
 
 .PHONY: test
 test:
-	SKIP_SQUASH=$(SKIP_SQUASH) hack/build.sh $(OS)-candidate
+	SKIP_SQUASH=$(SKIP_SQUASH) TEST_MODE=true hack/build.sh $(OS)-candidate
```
Comment (bparees) [inline Makefile]: TEST_MODE=true does nothing for this hack/build.sh

File: hack/build.sh [hunk 2]
```diff
@@ -1,25 +1,52 @@
 #!/bin/bash -e
-# $1 - Specifies distribution - RHEL7/CentOS7
+# This script is used to build, test and squash the OpenShift Docker images.
+#
+# $1 - Specifies distribution - "rhel7" or "centos7"
+# $2 - Specifies the image version - (must match with subdirectory in repo)
+# TEST_MODE - If set, build a candidate image and test it
```
Comment (bparees) [inline hack/build.sh]: $2 and TEST_MODE are not valid parameters for this script, they don't do anything.

**Comment by bparees** [comment]
2 nits and needs squash

**Comment by bparees** [comment]
meanwhile 
[test]

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
@mfojtik jenkins does not love me.

**Comment by mfojtik** [comment]
@bparees nits fixed && squashed

**Comment by mfojtik** [comment]
[test] ?

**Pull request merged**